sci-physics/fastjet/metadata.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>sci-physics@gentoo.org</email>
+		<name>Gentoo Physics Project</name>
+	</maintainer>
+	<longdescription lang="en">
+	A software package for jet finding in pp and e+e- collisions. It includes fast native implementations of many sequential recombination clustering algorithms, plugins for access to a range of cone jet finders and tools for advanced jet manipulation. 
+	</longdescription>
+	<use>
+		<flag name="python"> Build python bindings</flag>
+		<flag name="contrib"> Build fastjet-contrib (fjcontrib)</flag>
+		<flag name="plugins"> Build all standard and C++ plugins </flag>
+		<flag name="cgal"> Use <pkg>sci-mathematics/cgal</pkg> instead of bundled algorithms</flag>
```

Comment:
```suggestion
		<flag name="contrib">Build fastjet-contrib (fjcontrib)</flag>
		<flag name="plugins">Build all standard and C++ plugins </flag>
		<flag name="cgal">Use <pkg>sci-mathematics/cgal</pkg> instead of bundled algorithms</flag>
```
python is a global flag, so you don't need to define it locally here